lib/namespace.ts

```diff
@@ -195,4 +195,8 @@ export class Namespace {
     const tokenProvider = new AadTokenProvider(credentials);
     return Namespace.createFromTokenProvider(host, tokenProvider, options);
   }
+
+  public static getDeadLetterQueuePathForQueue(queueName: string): string {
```

Comment:
in TS all methods on the class are public by default. Hence there is no need to state that. Have been following this convention in all the projects.

Reply:
Thanks! Fixed.